setup/azurecmdfiles.wxi

```diff
@@ -20,12 +20,57 @@
                         <Component Id="cmp60B5B5F284C751DA655B0AB17F6C0528" Guid="*">
                             <File Id="fil7C0DE388E1197BC679F7D908CF3C01BF" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\Azure.AnalysisServices\Azure.AnalysisServices.psm1" />
                         </Component>
+                        <Component Id="cmp53002CE3F1D5F8C74FA6A0D73067CAAF" Guid="*">
```

Comment:
@cormacpayne Are you referring to the last bullet in the [guide](https://github.com/Azure/azure-powershell/wiki/Azure-Powershell-Developer-Guide#updating-the-installer)? I didn't look at the changes to the Wxi file, because I wouldn't know  what I'm looking for to be valid/invalid. I don't know Wxi files (not really something I'd assume most people are familiar with). It would probably make sense to **require a clean repo** prior to running the process instead of making a conditional check.